components/counter-bar/src/index.js

```diff
@@ -267,7 +267,7 @@ CounterBar.Counter.displayName = 'Counter';
 const sharedPropTypes = {
   active: PropTypes.bool,
   children: PropTypes.node,
-  component: PropTypes.any,
+  component: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
```

Comment:
was going to comment about what happens if it's a class-based component, but apparently `PropTypes.func` handles that :grin:

Reply:
Yes, `func` is the only one it would accept there, actually. Only the combination of `func` and `string` (or `any`!) is acceptable. I use the `Link` class component in the stories :)